src/Illuminate/Support/Env.php

```diff
@@ -200,17 +192,15 @@ public static function array(string $key, $default = null): array
     {
         $value = Env::get($key, $default);
 
-        if (is_null($value)) {
+        if ($value === null) {
             return [];
         }
 
         if (is_string($value)) {
             return array_map('trim', explode(',', $value));
         }
 
-        check_type($value, 'array', $key, 'Environment');
-
-        return $value;
+        return check_type($value, 'array', $key, 'Environment');
```

Comment:
If you want, even this can be shortened:
```suggestion
        return match (true) {
            $value === null => [],
            is_string($value) => array_map('trim', explode(',', $value)),
            default => check_type($value, 'array', $key, 'Environment'),
        };
```

Reply:
Thank you 😉 Already change💪

Reply:
Awesome! Thanks a lot 👍🏻